api/src/main/resources/messages.properties

```diff
@@ -87,6 +87,7 @@ error.patientProgram.enrolledDateEmpty=Enrolled date is empty, Please select a e
 error.patientProgram.enrolledDateDateCannotBeInFuture=Enrolled date can't be in future, Please select a date on or before today
 error.patientProgram.completionDateCannotBeInFuture=Completed date can't be in future, Please select a date on or before today
 error.context.null=contextDAO is null
+error.obs.encounter.mismatch=person objects of the obs and the encounter do not match
```

Comment:
Can you please capitalise the first letter?

Reply:
@wluyima I made the changes. Sorry for the late replys